app/screens/sso/sso.js

```diff
@@ -94,10 +94,14 @@ class SSO extends PureComponent {
     }
 
     componentDidMount() {
-        InteractionManager.runAfterInteractions(() => {
+        InteractionManager.runAfterInteractions(this.clearPreviousCookies);
+    }
+
+    clearPreviousCookies = () => {
```

Comment:
Should we clear these when logged out as well?

Reply:
The only way to access this screen is by being logged out

Reply:
Aah. Ok got it. Thanks.